templates/components/search/controls.html.twig

```diff
@@ -222,6 +222,19 @@ $(document).ready(function() {
       });
    });
 
+   $('.default-filter').change(function(event) {
+      var url = window.location.href;
+      url = url.replace(/&?nodefault/g, '');
+      if (!$(this).is(":checked")) {
+         if (url.indexOf('?') == -1)
+            url += '?nodefault';
+         else
+            url += '&nodefault';
+      }
+      url = url.replace(/\?$/g, '');
+      window.location.replace(url);
```

Comment:
Using regex does not seems safe. Using something like the following code would be more resilient to changes in URLs:
```suggestion
      const search_params = new URLSearchParams(window.location.search);
      if (!$(this).is(":checked")) {
          search_params.set('nodefault', '1');
      } else {
          search_params.delete('nodefault');
      }
      window.location.replace('?' + search_params.toString());
```